opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java

```diff
@@ -96,6 +96,30 @@ public void onAfterConcurrentHandlingStarted(HttpServletRequest httpServletReque
     };
 
     /**
+     * Decorator to record operation name, used in conjunction with Web Servlet Tracing Filter to provide tags.
+     */
+    HandlerInterceptorSpanDecorator STANDARD_OPERATION = new HandlerInterceptorSpanDecorator() {
```

Comment:
This name and javadoc do not say what is the operation name and javadoc here does not have to tell anything about tags added by the filter. Something like `HANDLER_METHOD_OPERATION_NAME` would be better.